### DISCUSSION TITLE
[IDEA/REQUEST] Writing class

### DISCUSSION THREAD

**Discussion by kmaustral** [jdanielmourao/obsidian-sanctum#88] [Ideas]
I'm used to writing Google Docs with a setting for space between paragraphs. This helps visualise units of text.  A global CSS for this is: 

.cm-s-obsidian .CodeMirror-line,
.cm-s-obsidian .cm-line {
    padding-bottom: 10px ;
    padding-top: 10px ;
}

But this is not appropriate for most files in the vault, particularly those using outlines.

I like your `cssclass: admonitions` in the YAML which helps style particular bullets. It would be great to have one like `cssclass: writing` which would add space between paragraphs in both preview and edit mode. Even better it would be good to have some ability to customise this space.

**Comment by jdanielmourao** [comment]
Hi! Thanks for the suggestion, I'll leave this open in case anyone else wishes to support it. I don't know if I'll add it for the time being, but I'll consider it for a future update if it garners enough support. :)

**Comment by jdanielmourao** [comment]
I've converted this to a discussion. :)

**Comment by kmaustral** [comment]
When I posted about this on Discord, Dor responded with a feature request: https://forum.obsidian.md/t/enter-new-paragraph-as-default/29361/17

I hadn't realised that paragraph was a feature of markdown. I've since started using Typora for writing, which differentiates between paragraph and line. That's relatively easy to integrate with Obsidian and helps focus on writing as a separate activity. 

A separate class for writing would still be welcome, but it's not so urgent.
